src/pilot/ironic.patch

```diff
@@ -18,12 +111,12 @@
  
  # Seconds to sleep between node lock attempts. (integer value)
  #node_locked_retry_interval = 1
-@@ -1432,7 +1432,7 @@
+@@ -1432,7 +1437,7 @@
  # doing the cleaning. If the timeout is reached the node will
  # be put in the "clean failed" provision state. Set to 0 to
  # disable timeout. (integer value)
 -#clean_callback_timeout = 1800
 +clean_callback_timeout = 5400
  
  # Timeout (seconds) to wait for a callback from the rescue
- # ramdisk. If the timeout is reached the node will be put in
+ # ramdisk. If the timeout is reached the node will be put in
```

Comment:
What is the no newline at the end of the file mean?  Never seen the circle with the - in it before.

Reply:
My IDE removes trailing lines at the end of a file,  I'll turn that off as I think Python wants a blank line, nothing critical.